management-api-server/src/test/java/com/datastax/mgmtapi/K8OperatorResourcesTest.java

```diff
@@ -2111,4 +2108,16 @@ public void testMoveAsync_MissingNewToken() throws Exception {
     verify(context.cqlService, never())
         .executePreparedStatement(any(), eq("CALL NodeOps.move(?, ?)"), eq("1234"), eq(true));
   }
+
+  static class Context {
+
+    Dispatcher dispatcher;
+    CqlService cqlService;
+
+    MockHttpResponse invoke(HttpRequest request) {
+      MockHttpResponse response = new MockHttpResponse();
+      dispatcher.invoke(request, response);
+      return response;
+    }
+  }
```

Comment:
Issue:
No need to move the inner class definition down. It just clutters up the diff

Reply:
I just realized this might be due to IDE settings. You can refer to [this](https://github.com/k8ssandra/management-api-for-apache-cassandra/#using-checkstyle-in-an-ide) for the Googe Java format style

Reply:
I've installed the google style plugin, hopefully everything is formatted better now.